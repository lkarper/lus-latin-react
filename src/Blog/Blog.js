import React, { Component } from 'react';

class Blog extends Component {
    render() {
        return (
            <main>
                <h2>Blog</h2>
                <article>
                    <h3>Est labore laborum ipsum culpa elit quis.</h3>
                    <p>Nisi enim adipisicing elit fugiat amet fugiat sunt. Ex elit adipisicing cillum anim ea cupidatat deserunt aute excepteur nulla exercitation sint ullamco in. Anim mollit id aliquip fugiat ea incididunt culpa commodo consectetur esse exercitation anim. Duis reprehenderit ex exercitation irure id enim commodo reprehenderit duis ipsum. Labore non exercitation proident ullamco mollit minim cupidatat excepteur ut ut laboris adipisicing.</p>
                </article>
            </main>
        );
    }
}

export default Blog;