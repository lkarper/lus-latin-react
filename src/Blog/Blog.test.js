import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Blog from './Blog';

describe('Blog component', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<Blog />, div);
        ReactDOM.unmountComponentAtNode(div);
    });

    it('renders the UI as expected', () => {
        const wrapper = shallow(<Blog />);
        expect(toJSON(wrapper)).toMatchSnapshot();
    });
});