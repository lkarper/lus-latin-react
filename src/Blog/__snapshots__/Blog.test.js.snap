// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blog component renders the UI as expected 1`] = `
<main>
  <h2>
    Blog
  </h2>
  <article>
    <h3>
      Est labore laborum ipsum culpa elit quis.
    </h3>
    <p>
      Nisi enim adipisicing elit fugiat amet fugiat sunt. Ex elit adipisicing cillum anim ea cupidatat deserunt aute excepteur nulla exercitation sint ullamco in. Anim mollit id aliquip fugiat ea incididunt culpa commodo consectetur esse exercitation anim. Duis reprehenderit ex exercitation irure id enim commodo reprehenderit duis ipsum. Labore non exercitation proident ullamco mollit minim cupidatat excepteur ut ut laboris adipisicing.
    </p>
  </article>
</main>
`;
